attribute #[using] for Definition and Fixpoint (#13183)
coq/coq: doc/sphinx/proof-engine/proof-handling.rst

      Use section variables in the forward transitive closure of the collection.
      The ``*`` operator binds stronger than ``+`` and ``-``.

   .. attr:: using

      This attribute can be applied to :cmd:`Definition`, :cmd:`Example`,
      :cmd:`Fixpoint`, :cmd:`CoFixpoint` and :n:`@thm_token` and carries

>       a :n:`@collection`, between quotes, as its value. The effect if
>       the same of one obtains by specifying the same :n:`@collection` to

Reviewer: ```suggestion
      a :n:`@collection`, between quotes, as its value. The effect is
      the same that one obtains by specifying the same :n:`@collection` to
```
